### ISSUE TITLE
Hadoop download link not working in install-hdfs.sh

### ISSUE THREAD

**Issue by laeeq80** [mcapuccini/SparkNow#7]
I will do a pull request later on but If I don't, this will become future pointer.

**Comment by mcapuccini** [comment]
Yep... a master student that is working on this had found this and other issues. He'll patch it.
